src/zeit/content/cp/layout.py

```diff
@@ -52,10 +52,12 @@ class ITeaserBlockLayout(zope.interface.Interface):
         default=1)
     areas = zope.schema.Set(
         title=u'Kinds of areas where this layout is allowed')
-    default_in_areas = zope.schema.Bool(
+    default_in_areas = zope.schema.Set(
         title=u'Kinds of areas where this layout is the default')
     types = zope.schema.Set(
         title=u'Types of CP where this layout is allowed')
+    leader = zope.schema.Bool(
```

Comment:
Es scheint mir hier eigentlich eher ein `is_leader` zu sein -- was zur Verwendung oben passen würde -- oder `needs_leader`, was eher zum title passt. Vielleicht ist es ja auch beides trifft ja auch beides zu... in der xml scheint es ja nur einmal benutzt zu werden, aber eigentlich ist ja genau die Konfigurierbarkeit angedacht, oder?